src/common/service-config/index.ts

```diff
@@ -31,6 +24,9 @@ export default class ServiceConfig {
   language?: string;
   subscriptions?: ServiceSubscriptions;
   debug?: string;
+  host?: string;
+  version?: string;
```

Comment:
Trying to get away from hardcoded versions inside service configs, so let's remove this